.rhdh/scripts/installation.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash  -e
+#
+# Copyright (c) 2024 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# install a helm chart with the correct global.clusterRouterBase
+
+# default namespace if none set
+namespace="rhdh-helm"
+chartrepo=0 # by default don't create a new chart repo unless the version chart version includes "CI" suffix
+github=0 # by default don't use the Github repo unless the chart doesn't exist in the OCI registry
+
+usage ()
+{
+  echo "Usage: $0 CHART_VERSION [-n namespace]
+
+Examples:
+  $0 1.1.1 
+  $0 1.7-20-CI -n rhdh-ci
+
+Options:
+  -n, --namespace   Project or namespace into which to install specified chart; default: $namespace
+      --github-repo If set will use the deprecated github repository to install the helm chart instead of the OCI registry.
+  -r, --chartrepo   If set, a Helm Chart Repo will be applied to the cluster, based on the chart version.
+                    If CHART_VERSION ends in CI and --github-repo is set, this is done by default.
+"
+  exit
+}
+
+if [[ $# -lt 1 ]]; then usage; fi
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-r'|'--chartrepo') chartrepo=1;;
```

Comment:
Since this option probably won't be used very often I would drop the `-r` short flag for simplicity.